core/state/snapshot/snapshot_test.go

```diff
@@ -119,7 +119,7 @@ func TestDiskLayerExternalInvalidationFullFlatten(t *testing.T) {
 	}
 	// Since the base layer was modified, ensure that data retrievals on the external reference fail
 	if acc, err := ref.Account(common.HexToHash("0x01")); err != ErrSnapshotStale {
-		t.Errorf("stale reference returned account: %#x (err: %v)", acc, err)
+		t.Errorf("stale reference returned account: %+v (err: %v)", acc, err)
```

Comment:
why change these?

Reply:
`go vet` was stopping it from compiling because the new `StateAccountExtra` type can't be formatted as hex. Surprised me too!

Reply:
and the original StateAccount can? perhaps this should be patched in upstream?

Reply:
> and the original StateAccount can?
Yeah because everything is just a `[]byte` or a `[32]byte`.

> perhaps this should be patched in upstream?
I'll do some experiments to find the correct formatting verb that preserves hex while also allowing us to add the payload.

Reply:
TIL about the `fmt.Formatter` interface, which means I can revert these changes as it will make `StateAccountExtra` compatible with any verb.